src/software/jetson_nano/display/screens/wheels_screen.py

```diff
@@ -0,0 +1,176 @@
+                cursor_pos_y = BASE_Y + self.font_size * self.curr_action
+            else:
+                cursor_pos_y = self.lcd_display.height - self.font_size - PADDING
+
+            self.lcd_display.draw.text(
+                (cursor_pos_x, cursor_pos_y), cursor, font=self.font, fill="#ffffff"
+            )
+
+            # x and y coordinates for drawing on screen
+            x = cursor_size
+            y = BASE_Y
+            en = True if self.actions[ENABLE_INDEX]["value"] else False
+
+            set_wheel_speed_str = "Set Wheel Speed: "
+            self.lcd_display.draw.text(
+                (x, y), set_wheel_speed_str, font=self.font, fill="#ffffff"
+            )
+            x += self.font.getsize(set_wheel_speed_str)[0]
+            self.lcd_display.draw.text(
+                (x, y),
+                "{}".format(en),
+                font=self.font,
+                fill="#00ff00" if en else "#0000ff",
+            )
+
+            x = cursor_size
+            y += self.font_size
+            front_left_str = "Front Left: "
+            self.lcd_display.draw.text(
+                (x, y), front_left_str, font=self.font, fill="#ffffff"
+            )
+            x += self.font.getsize(front_left_str)[0]
+            self.lcd_display.draw.text(
+                (x, y),
+                str(round(self.actions[FL_INDEX]["value"], 1)),
+                font=self.font,
+                fill="#00ffff",
+            )
+
+            x = cursor_size
+            y += self.font_size
+            front_right_str = "Front Right: "
+            self.lcd_display.draw.text(
+                (x, y), front_right_str, font=self.font, fill="#ffffff"
+            )
+            x += self.font.getsize(front_right_str)[0]
+            self.lcd_display.draw.text(
+                (x, y),
+                str(round(self.actions[FR_INDEX]["value"], 1)),
+                font=self.font,
+                fill="#00ffff",
+            )
+
+            x = cursor_size
+            y += self.font_size
+            back_left_str = "Back Left: "
+            self.lcd_display.draw.text(
+                (x, y), back_left_str, font=self.font, fill="#ffffff"
+            )
+            x += self.font.getsize(back_left_str)[0]
+            self.lcd_display.draw.text(
+                (x, y),
+                str(round(self.actions[BL_INDEX]["value"], 1)),
+                font=self.font,
+                fill="#00ffff",
+            )
+
+            x = cursor_size
+            y += self.font_size
+            back_right_str = "Back Right: "
+            self.lcd_display.draw.text(
+                (x, y), back_right_str, font=self.font, fill="#ffffff"
+            )
+            x += self.font.getsize(back_right_str)[0]
+            self.lcd_display.draw.text(
+                (x, y),
+                str(round(self.actions[BR_INDEX]["value"], 1)),
+                font=self.font,
+                fill="#00ffff",
+            )
+
```

Comment:
I think we can turn this into a loop from the `actions` dictionary right? something like this:

~~~
            def draw_actions(actions, cursor, start_x, start_y):                                                                                                                                                                        
                cursor = ">"                                                                                                                                                                                                            
                cursor_size = self.font.getsize(cursor)[0]                                                                                                                                                                              
                x = cursor_size                                                                                                                                                                                                         
                y += self.font_size                                                                                                                                                                                                     
                                                                                                                                                                                                                                        
                for action in actions:                                                                                                                                                                                                  
                                                                                                                                                                                                                                        
                    self.lcd_display.draw.text(                                                                                                                                                                                         
                        (x, y), actions["screen name"],                                                                                                                                                                                 
                        font=self.font, fill="#ffffff"                                                                                                                                                                                  
                    )                                                                                                                                                                                                                   
                                                                                                                                                                                                                                        
                    x += self.font.getsize(action["screen name"])[0]                                                                                                                                                                    
                                                                                                                                                                                                                                        
                    self.lcd_display.draw.text(                                                                                                                                                                                         
                        (x, y), str(round(action["value"], 1)),                                                                                                                                                                         
                        font=self.font, fill="#00ffff",                                                                                                                                                                                 
                    )
~~~

and then it can be shared between all screens that have menus

Reply:
I have implemented this function in the screens class so that it is available for all screens that are made. Child screens are still able to pass a `draw_screen` function if they require a different format such as `HomeScreen`. The `draw_actions` function is currently being used by all the child classes except for `HomeScreen`